README.md

```diff
@@ -975,3 +977,11 @@ The `source_map_options` (optional) can contain the following properties:
   [codegen]: http://lisperator.net/uglifyjs/codegen
   [compressor]: http://lisperator.net/uglifyjs/compress
   [parser]: http://lisperator.net/uglifyjs/parser
+
+#### Support for `const`
+
+`const` in `uglify-js@2.x` has function scope and as such behaves much like `var`,
+unlike `const` in ES2015 (ES6) which has block scope.
+
+It is recommended to avoid using `const` for this reason as it will have undefined
```

Comment:
Done in https://github.com/mishoo/UglifyJS2/pull/1916/commits/e405f5a8e4c10bc92e5ee454a1f0a63299878ab1